README.md

```diff
@@ -681,6 +690,243 @@ in and set by the script. Example:
 FT_CLIENT_CPU_MASK=0x100 TEST_CASE=1 IPERF=true CURL=false ./test.sh
 ```
 
+### Hardware Offload Validation
+
+Hardware Offload Validation is used to determine whether a flow has been hardware
+offloaded by examining the RX/TX packet counters from `ethtool` on the
+VF representors. When enabled, `iperf3` is run, `ethtool` on the VF representor is run at
+the beginning and end of the `iperf3` duration, `tcpdump` on the VF representor is run
+during the `iperf3` duration, lastly a summary of the results is printed.
+
+```
```

Comment:
Make sure this is the latest output. I see the following in code but not here:
echo "Client Pod on Client Host VF Representor Results:" > "${HWOL_VALIDATION_FILENAME}"
echo "Client Pod on Server Host VF Representor Results:" >> "${HWOL_VALIDATION_FILENAME}"
echo "Server Pod on Server Host VF Representor Results:" >> "${HWOL_VALIDATION_FILENAME}"

Reply:
Done